requirements/pandapower.txt

```diff
@@ -1,79 +1,263 @@
+    # via -r requirements/dev.txt
+appdirs==1.4.4
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   virtualenv
+attrs==20.3.0
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   jsonschema
+awesome-slugify==1.6.5
+    # via -r requirements/dev.txt
+bcrypt==3.2.0
+    # via
+    #   -r requirements/dev.txt
+    #   paramiko
+cached-property==1.5.2
+    # via -r requirements/dev.txt
+certifi==2020.12.5
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   requests
+cffi==1.14.6
+    # via
+    #   -r requirements/dev.txt
+    #   bcrypt
+    #   cryptography
+    #   pynacl
+cfgv==3.3.0
+    # via
+    #   -r requirements/dev.txt
+    #   pre-commit
+chardet==3.0.4
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   requests
+click==7.1.2
+    # via
+    #   -r requirements/dev.txt
+    #   click-default-group
+    #   rq
+click-default-group==1.2.2
+    # via -r requirements/dev.txt
+colorlog==4.7.2
+    # via -r requirements/dev.txt
+cryptography==3.4.7
+    # via
+    #   -r requirements/dev.txt
+    #   pandapower
+    #   paramiko
+cycler==0.10.0
+    # via matplotlib
+distlib==0.3.1
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   virtualenv
+et-xmlfile==1.1.0
+    # via openpyxl
+fabric3==1.14.post1
+    # via -r requirements/dev.txt
+filelock==3.0.12
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   tox
+    #   virtualenv
+identify==2.2.13
+    # via
+    #   -r requirements/dev.txt
+    #   pre-commit
+idna==2.10
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   requests
+jsonschema==3.2.0
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+kafka-python==2.0.2
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+kiwisolver==1.3.1
+    # via matplotlib
+matplotlib==3.4.0
+    # via -r requirements/pandapower.in
+networkx==2.6.2
+    # via pandapower
+nodeenv==1.6.0
+    # via
+    #   -r requirements/dev.txt
+    #   pre-commit
+numpy==1.20.3
+    # via
+    #   -r requirements/dev.txt
+    #   matplotlib
+    #   pandapower
+    #   pandas
+    #   scipy
+openpyxl==3.0.7
+    # via pandapower
+packaging==20.7
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   pandapower
+    #   tox
+pandapower==2.7.0
+    # via -r requirements/pandapower.in
+pandas==1.3.2
+    # via pandapower
+paramiko==2.7.2
+    # via
+    #   -r requirements/dev.txt
+    #   fabric3
+pendulum==2.1.2
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+pillow==8.3.1
+    # via matplotlib
+plotly==5.2.1
+    # via -r requirements/dev.txt
+pluggy==0.13.1
+    # via
+    #   -r requirements/dev.txt
+    #   d3a-interface
+    #   tox
+pre-commit==2.14.0
+    # via -r requirements/dev.txt
+psutil==5.8.0
+    # via -r requirements/dev.txt
+py==1.9.0
+    # via
```

Comment:
I really wonder who came up with this default setting to distribute `via -r requirements/dev.txt cffi` into new lines. Must be somebody who is paid by the added number of lines.

Reply:
It was done to avoid git conflicts, since they are more likely to happen in long lines :D